blueprints/app/files/.eslintignore

```diff
@@ -16,3 +16,5 @@
 /.node_modules.ember-try/
 /bower.json.ember-try
 /package.json.ember-try
+
+!.*
```

Comment:
Can you put this under miscellaneous please

Reply:
Yes.